tests/tensorflow/accuracy_aware_training/test_keras_api.py

```diff
@@ -102,10 +102,10 @@ def __call__(self, results_dict: Dict) -> float:
      'final_compression_rate',
      'reference_final_metric'),
     (
-            ({'maximal_relative_accuracy_degradation': 30.0}, 0.9077, 0.214535),
-            ({'maximal_relative_accuracy_degradation': 1.0}, 0.3577, 0.210423),
-            ({'maximal_absolute_accuracy_degradation': 0.0}, 0.1202, 0.236087),
-            ({'maximal_absolute_accuracy_degradation': 0.10}, 0.9077, 0.108224),
+            ({'maximal_relative_accuracy_degradation': 30.0}, -0.0923, 0.04942),
```

Comment:
Model from this test is now initialized with different weights, this leads to new reference values. These values seem bad, but if seed is changed, they vary and might become ok. Decided not to change seed and just change the reference values in this PR. This test is in general badly designed, e.g. data is generated randomly and model can't learn properly. Will need to redesign this test.

Reply:
This doesn't look like a success story. What do you think about pre-trained or fixed weights for testing?

Reply:
Yes, it doesn't. But I think this should be addressed in a separate PR. Created a ticket for this: 108356.